README.md

```diff
@@ -2,15 +2,38 @@
 
 ## Introduction
 
-This repo provides the code for several example dApps in the [Axelar Local Development Environment](https://github.com/axelarnetwork/axelar-local-dev). Examples contain both JavaScript and Solidity smart contract code.
+Welcome to Axelar cross-chain dApp examples! This repository provides the code for several example dApps in the [Axelar Local Development Environment](https://github.com/axelarnetwork/axelar-local-dev). These examples contain both JavaScript and Solidity smart contract code that can be used as a starting point for building your own cross-chain dApps. This readme provides a detailed guide on how to set up the development environment and run the examples.
 
-**Note:** Some example folders in this repo are not documented.
+**Note:** Please be aware that some example folders in this repository are currently being documented. Our team is working diligently to provide comprehensive documentation for all examples to ensure that you have the best possible experience building with our tools.
+
+### Table of Contents
+- [Introduction](#introduction)
+- [Getting Started](#getting-started)
+  - [One-time setup](#one-time-setup)
+  - [Set environment variables](#set-environment-variables)
+  - [Print wallet balances](#print-wallet-balances)
+  - [Running the local chains](#running-the-local-chains)
+- [Examples](#examples)
+  - [Evm Examples](#evm-examples)
+    - [Basic](#basic)
+    - [Advanced](#advanced)
+  - [Aptos Examples](#aptos-examples)
+  - [Web Examples](#web-examples)
+
+
+## Introduction
+
+[Axelar Network](https://axelar.network/) is the universal interoperability network that securely connects all blockchain ecosystems, applications, assets, and users. Axelar cross-chain dApp examples demonstrate how to build cross-chain applications using Axelar SDK. In this repository, we provide several examples of dApps that demonstrate how to use Axelar to build cross-chain dApps.
+
+## Getting Started
+
+To build cross-chain dApps using Axelar SDK, you must first set up your development environment. This section provides a detailed explanation on how to set up your development environment for running the examples.
 
 ## One-time setup
 
-Install [nodejs](https://nodejs.org/en/download/). Run `node -v` to check your installation.
+Before you can start running the examples, you need to install [Node.js](https://nodejs.org/en/download/). You can check your Node.js version by running `node -v`. We support Node.js version 16.x and 18.x.
 
-Support Node.js version 16.x and 18.x
+Ensure you also have [python](https://www.python.org/) installed and configured on your machine. You can check your python version by running `python --version`. We support python version 3.8.x.
```

Comment:
Let's fix this so it's not a dependency. I've been talking to @npty about this.

Reply:
Okay. A developer raised this issue on Discord a few days ago, and then from the error, I figured the error indicates Python wasn't installed on his PC; that's why I updated the docs: See the screenshot below.

![Screenshot 2023-04-19 at 14 41 01](https://user-images.githubusercontent.com/45847909/233093471-e4797210-4984-4d3e-9bf6-bca6c5cc16a0.png)

cc: @npty

Reply:
@npty I didn't encounter the same error too. I got that complaint from one of the devs trying to set up the project locally for the first time.